ocs_ci/ocs/fio_artefacts.py

```diff
@@ -56,6 +56,8 @@ def get_mcg_conf(mcg_obj, workload_bucket, custom_options=None):
     # add or overwrite custom values
     if custom_options:
         for section in custom_options:
+            if section is not 'create':
```

Comment:
@tiffanyn108 
You can use ```config.has_section(section)``` function to verify if such a section exists or not then based on that take decision whether to add or set

Reply:
Acked, fixed it.